scripts/openmc-voxel-to-vtk

```diff
@@ -8,7 +8,6 @@ import numpy as np
 import h5py
 import vtk
 
-
```

Comment:
PEP8 actually specifies two blank lines before functions

Reply:
Oops. Thanks! On it.